osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -76,7 +76,7 @@ void genAddressesFromAddr(const struct ifaddrs* addr, QueryData& results) {
       r["point_to_point"] = dest_address;
     }
   }
-
+  r["type"] = "unknown";
```

Comment:
Need to update spec file which doesn't have unknown.

Reply:
maybe we could use other instead of unknown

Reply:
Agree with updating spec file, thanks.

Do not agree with using `other` for `unknown` for two reasons:
1. By meaning `other` means we know the type of interface, but the one is differ from `dhcp`, `auto`, `manual`.
2. Windows is already uses the word `unknown` see implementation for the windows: [ref](https://github.com/facebook/osquery/blob/master/osquery/tables/networking/windows/interfaces.cpp#L202).